_posts/en/newsletters/2020-09-02-newsletter.md

```diff
@@ -0,0 +1,243 @@
+    there be?  Under what circumstaces should the default signet be
+    reset?  When should proposed consensus changes to mainnet Bitcoin be
+    implemented by signet miners (trusted signers)?
+
+    Anyone interested in helping answer these questions is encouraged to
+    respond.
+
+- **Standardizing temporarily trusted LN channels:** Roei Erez
+  [posted][temporarily trusted channels] to the Lightning-Dev mailing list a proposal to standardize
+  the way LN implementations deal with new channels when the
+  participants are willing to trust an unconfirmed deposit transaction.
+  For example, Alice pays Bob's established business to fund a
+  channel to her.  In that case, Alice may be willing to trust Bob not
+  to double spend his deposit transaction.  Bob can also safely accept
+  in-channel payments from Alice because he gets all his money back
+  anyway if the deposit transaction doesn't confirm.
+
+    Erez noted that several LN implementations already allow users to
+    opt-in to trusting certain unconfirmed channels but that the
+    different implementations vary in some aspects, such as what
+    `short_channel_id` to use until the deposit transaction confirms
+    (this value usually identifies the location of the deposit
+    transaction in the block chain, which doesn't apply to unconfirmed
+    transactions).
+
+    Participants in the mailing list discussion seemed supportive of the
+    proposal.
+
+## Recently transcribed talks and conversations
+
+*[Bitcoin Transcripts][] is the home for transcripts of technical
+Bitcoin presentations and discussions. In this monthly feature, we
+highlight a selection of the transcripts from the previous month.*
+
+- **nix-bitcoin**: nixbitcoindev appeared on the Stephan Livera Podcast to
+  discuss nix-bitcoin.  nix-bitcoin is an experimental but promising project
+  that aims to improve the security of installing Bitcoin and Lightning nodes. It
+  uses the functional Linux distribution NixOS where upgrading is equivalent to
+  installing from scratch due to atomicity. Through minimalism, reproducibility,
+  compartmentalization and defense in depth nix-bitcoin aims to reduce the attack
+  surface of a Bitcoin and Lightning software stack with programs unaware of
+  other running processes unless they need to be. ([transcript][nixbitcoin
+  transcript], [audio][nixbitcoin audio])
+
+- **Taproot activation**: Eric Lombrozo, Luke Dashjr and Aaron van Wirdum
+  discussed the alternative [Taproot][topic taproot] activation proposals (see
+  [Newsletter #107][news107 taproot activation]) and gave their opinions in a
+  personal capacity on what lessons, if any, could be derived from activating the
+  SegWit soft fork.  Lombrozo and Dashjr both believe that the Taproot activation
+  process should not be drawn out and that all opposition, criticism or review of
+  the proposed code changes should be flushed out prior to starting any possible
```

Comment:
could "flushed out" be replaced with something clearer? it seems ambiguous or could be interpreted negatively

Reply:
Hmm I see what you mean. "should be obtained prior to"?

Reply:
"have rough consensus"? not sure what they meant

Reply:
The point is rather than choosing a long multiphase activation process giving miners lots of opportunity to signal opposition or demand changes/additional review you front load all those discussions before you start any activation process. The activation process should only begin once everyone (including the miners) are onboard. I thought it was a convincing argument but maybe I'm not doing it justice here.

Reply:
From the transcript:

"If it doesn’t feel it is decisive and there isn’t a decision that has been made then people are going to be totally confused. It is cause a lot of uncertainty. I think that this is something where we either need to be very aggressive about it and get everyone onboard and be decisive and say “Yes this is going to happen” or it is not worth doing at all. I don’t like this idea of seeing what happens in 6 months or whatever. We either decide right away that yes we are going to do it or no."

"It kind of invites controversy."

Reply:
> "With this kind of stuff I think people are looking for decisiveness and looking for resolution.

This is definitely what I hear from people.

Maybe "should be decided and resolved prior to..."  :D